cpp/include/Ice/Config.h

```diff
@@ -5,6 +5,8 @@
 #ifndef ICE_CONFIG_H
 #define ICE_CONFIG_H
 
+// IWYU pragma: keep
```

Comment:
what is this about?

Reply:
I'm testing with the `include-what-you-keep` tool to find unnecessary includes.